examples/oneapi/cpp/makefile_lnx

```diff
@@ -147,7 +147,7 @@ sointel64:
 _make_ex: $(RES)
 
 vpath
-vpath %.cpp $(addprefix ./source/,pca svm)
+vpath %.cpp $(addprefix ./source/,pca decision_forest svm)
```

Comment:
I think it's better to add by alphabetically

Reply:
fixed